packages/core/src/app/order/OrderStatus.tsx

```diff
@@ -74,6 +75,13 @@ const OrderStatus: FunctionComponent<OrderStatusProps> = ({
                             />
                         </p>
                     );
+                } else if (payment.mandate?.mandateText) {
```

Comment:
Can we move this whole logic from line 50 to another component? This is getting quite big slowly.